documentation/modules/oauth/proc-oauth-broker-config.adoc

```diff
@@ -7,19 +7,22 @@
 
 This procedure describes how to configure Kafka brokers so that the broker listeners are enabled to use {oauth} authentication using an authorization server.
 
+{oauth} requires configuration of TLS listeners.
```

Comment:
One thing is to say that something is not recommended and other is that it is not required. There are many valid reasons why user might choose to use OAuth over unencrypted interface and we support it. So IMHO we should stay with recomendations only.

Reply:
Good point.
We advise use of {oauth} over an encrypted interface through configuration of TLS listeners.
Plain listeners are not recommended.